aspnet/mvc/overview/older-versions/getting-started-with-aspnet-mvc4/adding-a-model.md

```diff
@@ -1,15 +1,15 @@
 ---
 uid: mvc/overview/older-versions/getting-started-with-aspnet-mvc4/adding-a-model
-title: "Adding a Model | Microsoft Docs"
+title: "Adding a Model 2012 | Microsoft Docs"
 author: Rick-Anderson
-description: "Note: An updated version of this tutorial is available here that uses ASP.NET MVC 5 and Visual Studio 2013. It's more secure, much simpler to follow and demo..."
+description: "In this section you'll add some classes for managing movies in a database. These classes will be the model part of the ASP.NET MVC app."
```

Comment:
In descriptions, we eliminate introductory phrases such as "In this tutorial" and get right to the point. "Add some classes for managing movies...."